Woof/Woof/ViewModel/OwnerProfileViewModel.swift

```diff
@@ -1,4 +1,5 @@
 import Foundation
+import SwiftUI
```

Comment:
We do not need import `Foundation`, if we import `SwiftUI`
```suggestion
import SwiftUI
```

Reply:
Fixed.